cmd/controller/root.go

```diff
@@ -320,6 +320,11 @@ var rootCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		if err := mgr.AddReadyzCheck("readyz", healthz.Ping); err != nil {
+			setupLog.Error(err, "unable to add controller readyz check")
+			os.Exit(1)
+		}
```

Comment:
There is no difference between healthz and readyz. What's the purpose of this then?